### ISSUE TITLE
Error popup followed by crash

### ISSUE THREAD

**Issue by wasted1** [processing/processing#4980]
![processing_tabs_error_25-03-2017](https://cloud.githubusercontent.com/assets/26678313/24325729/8da280ec-119f-11e7-827b-bca31b824926.png)

Attempted to save my project, I tend to mash cntrl-s while working on stuff. Noticed a stacktrace in the console. Attempted to save, save as, but couldn't save the project. The stacktrace named a specific tab or editor window. Copied the code from said tab and tried to delete the tab to recreate and save it. Got another stacktrace. Closed the project without saving. Got the attached error on launch. Can't open the project without getting this error. I can see the processing window behind it but can't edit it as the error forces its focus. Can't copy and add the stacktrace because I can't see it and can't increase the console size since the error blocks interaction with processing's window.

**Comment by benfry** [comment]
This error would happen if it couldn't read one of the files in the sketch. Is it possible that it disappeared? Or wasn't openable? Could you post a copy of the sketch?

**Comment by wasted1** [comment]
I don't think so. The initial errors I got while control+s'ing did mention being unable to save the specific tab but it was still in the directory. I managed to copy and paste all the code out and re-create the project's tabs in a new sketch. I don't have the original sketch anymore. It's been a month..
